README.md

```diff
@@ -1,15 +1,38 @@
 Tuxemon
 ==============
 
-Tuxemon is a free, open source monster-fighting RPG.  It's in constant
-development and improving all the time!  Contributors of all skill and
-level are welcome to join.
+Tuxemon is a free, open source monster-fighting role player game (RPG). It hosts a turn-based combat style along with monster acquisition features. 
```

Comment:
please don't change the formatting.  line length should be limited to 79 characters, unless it is part of markup (long hyperlinks, markdown, etc)